.github/workflows/test-postgresql.yml

```diff
@@ -57,5 +57,5 @@ jobs:
           FF_TEST_DB_POSTGRES_USER: postgres
           FF_TEST_DB_POSTGRES_PASSWORD: secret
           FF_TEST_DB_POSTGRES_DATABASE: flowforge
-          NODE_OPTIONS: "--max_old_space_size=4096"
+          NODE_OPTIONS: "--max-old-space-size=6144"
```

Comment:
AIUI this controls when GC kicks in, rather than being about allocating more memory to the process. The previous lower value would mean GC is kicking in earlier and reclaiming memory.

Reply:
As to why the process crashes with the GC runs 🤷